src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java

```diff
@@ -8471,8 +8471,112 @@ private void registerSolderingAlloyRecipes() {
                         GT_ModHandler.getModItem(Computronics.ID, "computronics.dockingUpgrade", 1L, 0),
                         250,
                         480);
+
+                {
+                    if (IndustrialCraft2.isModLoaded()) if (GraviSuite.isModLoaded()) if (GalacticraftMars
```

Comment:
ic2 should be a dep via gt, so unnecessary. the other two were probably supposed to apply to all these recipes? just do && and then dont forget the { }